test/extended/idling/idling.go

```diff
@@ -282,8 +282,10 @@ var _ = g.Describe("idling and unidling", func() {
 		})
 
 		g.It("should work with TCP (when fully idled) [Conformance] [local]", func() {
+			err := exutil.WaitForUserBeAuthorized(oc, oc.Username(), "create", "endpoints/restricted")
```

Comment:
Where is this permission being added?

Reply:
good question... i only see this added for 'system:controller:endpoint-controller', now I wonder how this test passed...

Reply:
@DirectXMan12 any idea? i can't find this anywhere in origin or kube codebase, but https://github.com/mfojtik/origin/blob/ff71dafe2ac70ec51d6f3122b01b058d1970dde6/pkg/service/admission/endpoint_admission.go#L82 use it.

Reply:
hmm... did something change recently about the user the tests were running on?  We've never had this issue before (the code in question just needs to annotate the endpoints objects).

Reply:
agree... I want to understand why this is needed now